src/views/Home.css

```diff
@@ -0,0 +1,53 @@
+* {
+	box-sizing: border-box;
+	margin: 0;
+	padding: 0;
+}
+.Layout-main {
+	margin: 0;
+	max-width: 100vh;
+	width: 100%;
+}
+.container,
+.Home {
+	display: flex;
+	flex-direction: column;
+	justify-content: center;
+	align-items: center;
+}
+.container {
+	margin: 10px;
+}
+#joinList {
+	padding: 10px 10px 8px 10px;
+	box-shadow: 3px 1px 5px 1px rgba(211, 211, 211, 0.75);
+	-webkit-box-shadow: 3px 1px 5px 1px rgba(211, 211, 211, 0.75);
+	-moz-box-shadow: 3px 1px 5px 1px rgba(211, 211, 211, 0.75);
+	border-radius: 2px;
+}
+.welcome {
+	text-transform: uppercase;
+	font-weight: bold;
+	text-shadow: rgba(0, 0, 0, 0.15) 0 1.5px;
+	margin: 15px 0;
+}
+.actions {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-around;
+	align-items: center;
+	margin: 10px 20px;
+	padding: 10px 0;
+}
+.homeImages {
+	width: 100%;
+	margin: 10px 0;
+}
+#homeImg {
+	height: 55vh;
```

Comment:
It looks like you merged in the branches @a-cole55 and I were working on. I believe you may have an older version of Ar'Niqua's branch as the `height: 55vh;` is something that was addressed. Not quite sure how to handle that; but there may be a fair number of merge conflicts to address.